Surface_mesh_simplification/doc/Surface_mesh_simplification/Surface_mesh_simplification.txt

```diff
@@ -209,20 +209,31 @@ But this is a trade-off: the cost of a collapse is a function of the placement
 (the new position chosen for the remaining vertex) so before `GetCost`
 is called for each and every edge, `GetPlacement` must also be called to obtain
 the placement parameter to the cost function.
-But that placement, which is a 3D Point, is not attached to each and every edge since
+But that placement, which is a 3D point, is not attached to each and every edge since
 that would easily <I>triple</I> the additional storage requirement.
 On the one hand, this dramatically saves on memory but on the other hand is
 a processing waste because when an edge is effectively collapsed, `GetPlacement`
 must be called <I>again</I> to know were to move the remaining vertex.
 
-Earlier prototypes shown that attaching the placement to the edge, thus avoiding one
+Earlier prototypes showed that attaching the placement to the edge, thus avoiding one
 redundant call to the placement function after the edge collapsed, has little
 impact on the total running time. This is because the cost of an each edge is not just
 computed once but changes several times during the process so the placement function
 must be called several times just as well. Caching the placement can only avoid the
 very last call, when the edge is collapsed, but not all the previous calls which
 are needed because the placement (and cost) changes.
 
+We have not yet explained what the `Filter` policy is about.
```

Comment:
Strange sentence

Reply:
The expression exists (see [here](https://www.linguee.fr/anglais-francais/traduction/not+yet+explained.html)), but maybe you propose something like "The last parameter....."?

Reply:
Yes something like "The Filter policy allows to ...."

Reply:
changed.